src/FAQ/FaqData.js

```diff
@@ -1,40 +1,49 @@
-     link: <a className="color-navigation font-weight-bold"target="blank"href="https://levelup.charite.de/app-demo">https://levelup.charite.de/app-demo</a>
-},
-{
-    question: 'Kann ich LevelUp vom Handy aus nutzen?',
-    answer: 'Ja, LevelUp lässt sich ganz bequem  vom Handy aus nutzen. Einfach im Handy-Browser die Adresse aufrufen. LevelUp ist keine App, sondern wird im Browser dargestellt. Somit wird kein Speicherplatz für die App auf Deinem Handy benötigt.'
-},
-{
-    question: 'Meine letzten Prüfungen werden nicht dargestellt, warum nicht?',
-    answer: 'Die Ergebnisse der Prüfungen werden erst nach einigen Wochen in LevelUp dargestellt. Teilweise kann es leider auch einige Monate dauern, wie beim PTM. Nicht davon betroffen sind die Daten aus HIS wie Anwesenheiten und Meilensteine wie Pflegepraktikum etc. Diese werden spätestens 2 Tage nach der Darstellung in HIS auch in LevelUp angezeigt.'
-},
-{
-    question: 'Eines meiner Prüfungsergebnisse stimmt nicht mit den Daten im HIS überein. Woran liegt das?',
-    answer: 'Möglicherweise handelt es sich um einen Importfehler. Bitte kontaktiere in diesem Fall den Support unter',
-    email :<a className="color-navigation font-weight-bold"href="mailto:levelup@charite.de">levelup@charite.de</a>
-},
-{
-    question: 'Ich kann mich mit meinen Charité-Logindaten im LevelUp nicht anmelden/registrieren.',
-    answer:'Bist Du aktuell im MSM 2.0 immatrikuliert? Wenn ja,  kontaktiere den Support unter',
-    email: <a className="color-navigation font-weight-bold"href="mailto:levelup@charite.de">levelup@charite.de</a>
-}
+    {
+        question: 'Was ist LevelUp und wofür brauche ich das Tool?',
+        answer: 'LevelUp ist ein neues Tool für alle Studierenden im MSM 2.0 an der Charité. Du siehst hier sämtliche Prüfungsergebnisse und deinen Lernfortschritt im Studium. Es zeigt deine MC-Prüfungsergebnisse bis ins Detail, inkl. Fragen, Antworten und Lösungen. Die praktischen ärztlichen Tätigkeiten (EPAs), die im Studium gelernt werden, werden hier dokumentiert und bewertet, sowohl durch dich selbst, als auch deine Dozierenden. Du kannst außerdem deinen Wissenszuwachs anhand des PTM-Features verfolgen. Dein Wissen wird auch nach Fachgebiet gerankt und du siehst wo aktuell deine Stärken sind, und wo du noch üben musst.'
+    },
+    {
+        question: 'Wie nutze ich LevelUp?',
+        answer: <Fragment><a className="color-navigation font-weight-bold" href="https://levelup.charite.de/app/tutorial/src/index.html">Hier</a> geht es zum Tutorial für das Tool, und <a className="color-navigation font-weight-bold" href="">Hier</a> geht es zu einem kurzen Clickvideo, in dem gezeigt wird, wie man auf der Seite navigiert.</Fragment> 
```

Comment:
Hm - ich finde, hier hat das `Fragment` nichts zu suchen. Diese Datei ist so schön strukturiert, question und answer sind immer einfach strings. Diese Struktur zu brechen, nur um per `Fragment` das Einfügen eines `div` zu verhindern, finde ich architektonisch einfach falsch. 

Bitte mach es ähnlich wie in Zeile 22 bei "Kann ich Levelup ansehen, wenn ich nicht studiere?". Nur statt eines einzelnen links machst du eben `span` oder `p` oder `div` mit zwei Links drin.

Reply:
Mit "Fragment" war nur vorübergehende Lösung, weil ich die Datei im Tickt "legendeText _ 2557" komplett refrakturieren wollte. Ich hab es jetzt in diesem Tickt gemacht. Jetzt enthält die Datei nur Strings